2280 python function readcfg not working when multiple keys are given

prudhomm opened the pull request:
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/feelpp/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes?
- [x] Have you successfully run the Feel++ testsuite with your changes locally?
- [ ] Have you written Doxygen comments in your contribution ?

-----

- closes #2280 
- #2016 
- #1494

prudhomm committed:
use cln from spack environment

add support for spack #2016

prudhomm committed:
update external cln support based on pkgconfig

add support for spack #2016
Clean Code #1494

prudhomm committed:
we do not use mpi_cxx

Clean Code #1494

prudhomm committed:
fix configparser to enable keys with multiple values

#2280

prudhomm committed:
install rpath when using spack

add support for spack #2016

thomas-saigre commented on python/pyfeelpp/feelpp/__init__.py.in:
```diff
@@ -32,13 +32,24 @@ def readCfg(filename):
+                    return # ignore conversion list to string (line 554)
+            super(MultiOrderedDict, self).__setitem__(key, value)
+
+    config = ConfigParser.RawConfigParser(dict_type=MultiOrderedDict, strict=False)
```
@prudhomm it seems that `RawConfigParser` is not a member of the class `ConfigParser`, but a class of the library `configparser` (cf [the documentation](https://docs.python.org/3.10/library/configparser.html#rawconfigparser-objects)).
When I run the test locally, I get this error :
```sh
3: >       config = ConfigParser.RawConfigParser(dict_type=MultiOrderedDict, strict=False)
3: E       AttributeError: type object 'ConfigParser' has no attribute 'RawConfigParser'
3: 
3: /data/home/saigre/develop/install/default/lib/python3.10/site-packages/feelpp/core/__init__.py:54: AttributeError
3: =========================== short test summary info ============================
3: FAILED ../../../../../python/pyfeelpp/tests/test_core.py::test_config_parser - AttributeError: type object 'ConfigParser' has no attribute 'RawConfigParser'
```
Oddly, in the PR the test appear to have passed, but they actually failed : https://github.com/feelpp/feelpp/actions/runs/10657011721/job/29546848919

thomas-saigre committed:
fix import configParser #2280 #2281
/cc @prudhomm
[skip feelpp, skip tests, skip test,  skip toolboxe, skip toolboxes, skip mor]

prudhomm commented:
@vincentchabannes PR is ready
